admin_manual/configuration/user/user_provisioning_api.rst

```diff
@@ -245,6 +245,100 @@ XML Output
     <data/>
   </ocs>
 
+Enable User
+-----------
+
+Enable a user.
+Authentication is done by sending a Basic HTTP Authorization header.
```

Comment:
Could you link "Basic HTTP Authorization header" to the documentation on the topic?